front-end/src/app/cruds/pacientes/paciente-listar/paciente-listar.component.ts

```diff
@@ -38,18 +38,20 @@ export class PacienteListarComponent implements OnInit {
       icon: 'pi pi-info-circle',
 
       accept: () => {
-        this.pessoaService.deletaPaciente(id).subscribe(paciente => { this.msgs = [];
+        this.pessoaService.deletaPaciente(id).subscribe(paciente => {
+          this.msgs = [];
           this.msgs = [{ severity: 'info', summary: 'Concluído', detail: 'Registro Excluido' }];
-          this.confirmationService.close();
-          location.reload; },
-           err => { console.log('Erro ao deletar paciente') });
-        
+          location.reload;
+        },
+          err => { console.log('Erro ao deletar paciente') });
```

Comment:
FDP pela terceira vez tira essa bosta daqui e poe o `this.msgs = [{ severity: 'error', summary: 'Erro', detail: error.error }]